documentation/asciidoc/topics/proc_xsite_disable_gossip_router.adoc

```diff
@@ -0,0 +1,47 @@
+[id='xsite-disable-gr_{context}']
+= Disable local Gossip Router and service
+
+[role="_abstract"]
+Only a single Gossip Router is required to route the cross-site traffic however, the {brandname} operator starts a Gossip Route on each site.
+You can disable the Gossip Router in one of the sites to save resources.
```

Comment:
What's the best practice if 3 sites are configured?

If you have a GossipRouter enabled on site 1, but not 2 & 3, does that mean that xsite will fail if Site 1 becomes unavailable?

Reply:
the best practice for 2 or more sites is to let the operator start the Gossip Router on all sites. 

This is a customer request that does not want to have a Gossip Router on one of the sites (2 sites total).

In your scenario, the connection will be broken and cross-site becomes available.

Reply:
Should we add a note explaining this to the docs, maybe as a NOTE section?

Reply:
+1, I'll leave this for the @infinispan/docs team :)

Reply:
Cool. I'm looking into it and I'll be back on Monday